builders/integrations/indexers/thegraph.md

```diff
@@ -233,35 +231,33 @@ export function handlePlayerJoined(event: PlayerJoined): void {
     player = new Player(playerId);
   }
   player.address = event.params.player;
-
   player.save();
-
   // ID for the ticket (round - player_address - ticket_index_round):
   // "round_number" + "-" + "player_address" + "-" + "ticket_index_per_round"
```

Comment:
this line can be deleted

Reply:
this is displaying weird, but this line is the one that can be deleted `  // "round_number" + "-" + "player_address" + "-" + "ticket_index_per_round"`